hide/comp/SceneEditor.hx

```diff
@@ -1337,6 +1396,32 @@ class SceneEditor {
 		saveDisplayState();
 	}
 
+	public function setLock(elements : Array<PrefabElement>, unlocked: Bool) {
+		for(o in elements) {
+			if(unlocked) {
+				for(c in o.flatten(Object3D)) {
+					lockList.remove(c);
+					var el = tree.getElement(c);
+					applyTreeStyle(c, el);
+					applySceneStyle(c);
+				}
+			}
+			else {
+				lockList.set(o, true);
+				var el = tree.getElement(o);
+				applyTreeStyle(o, el);
+				applySceneStyle(o);
```

Comment:
`Prefab.flatten` already contains `this` so it looks like you don't need this

Reply:
Fixed